src/App.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {
+  Routes, Route, Link,
+} from 'react-router-dom';
+import Book from './Components/Books';
+import AddBook from './Components/Form';
+
+const NavBar = () => (
+  <nav className="NavBar">
+    <h2 className="NavTitle">Booklist</h2>
+    <Link to="/">Books</Link>
+    <Link to="/categories">Categories</Link>
+  </nav>
+);
+
+const Books = () => (
+  <div className="HomePage">
+    <ul className="Booklist">
+      <Book />
+      <Book />
+    </ul>
+    <AddBook />
+  </div>
+);
+
+const Categories = () => (
+  <div>
+    <ul>
+      <h3>Categories</h3>
+      <li>All</li>
+    </ul>
+  </div>
+);
```

Comment:
- [OPTIONAL] You can put these functions into a separate files as components.

Reply:
I will put each page in a different file. Thank you.